admin-cli/build.gradle

```diff
@@ -7,6 +7,8 @@ dependencies {
     implementation project(':client')
     implementation project(':common')
     implementation 'com.networknt:json-schema-validator:1.0.73'
+    implementation 'org.springframework.boot:spring-boot'
```

Comment:
I think `spring-boot-starter-web` contains `spring-boot` so you don't need both. You can use `gradle admin-cli:dependencies` to see the dependency graph for this subproject.